mlflow/tracing/clients/ipython_wrapper.py

```diff
@@ -0,0 +1,42 @@
+from trace import Trace
+
+from mlflow.tracing.clients.base import TraceClient
+from mlflow.tracing.clients.local import InMemoryTraceClient
+
+
+class IPythonTraceClient(InMemoryTraceClient):
```

Comment:
@B-Step62 does this look okay? i'm not sure if creating a wrapper is the best way, but I felt it would be convenient later to swap out the client.

Reply:
@daniellok-db I'm actually wondering if we need to keep the original `InMemoryTraceClient`...? It was particularly created for notebook demo.

Reply:
It's a bit concerning to extend singleton class... given its mechanism relies on class instance. What about adding the displaying logic in `InMemoryTraceClient` and gate it behind `if IPython.get_ipython() is not None` condition?

Reply:
makes sense!